pilot/pkg/model/authentication.go

```diff
@@ -15,13 +15,24 @@
 package model
 
 import (
+	"crypto/sha1"
 	"fmt"
+	"net/url"
+	"strconv"
+
+	"github.com/gogo/protobuf/types"
 
 	authn "istio.io/api/authentication/v1alpha1"
 	meshconfig "istio.io/api/mesh/v1alpha1"
+	mccpb "istio.io/api/mixer/v1/config/client"
 	"istio.io/istio/pkg/log"
 )
 
+const (
+	// Defautl cache duration for JWT public key.
+	jwtPublicKeyCacheSeconds = 60 * 5
```

Comment:
Why 5 min ? Typical JWT is 1 hour.

Is there a flag or env variable to override (if yes - maybe mention it in the comment :-) ?

Reply:
I don't know where did I pick up this value. Anyway, you are right, it should be in a config somewhere. Added comment for that. I'll revisit this later to adjust the constant.